batchflow/named_expr.py

```diff
@@ -16,30 +16,33 @@ def __init__(self, pipeline):
 
 def eval_expr(expr, **kwargs):
     """ Evaluate a named expression recursively """
-    if isinstance(expr, NamedExpression):
-        _expr = expr.get(**kwargs)
-        if isinstance(expr, W):
-            expr = _expr
-        elif isinstance(_expr, (NamedExpression, list, tuple, dict, Config)):
-            expr = eval_expr(_expr, **kwargs)
-        else:
+    try:
+        if isinstance(expr, NamedExpression):
```

Comment:
Perhaps `try...except` should be only within the first branch, as other branches are just for containers.